JsonSchema/ErrorMessages.cs

```diff
@@ -53,7 +53,7 @@ public static string ReplaceTokens(this string message, params (string token, ob
 		{
 			var parameter = parameters[i];
 #pragma warning disable IL2026, IL3050
-			values[i] = JsonSerializer.Serialize(parameter.value, JsonSchemaSerializerContext.OptionsManager.SerializerOptionsUnsafeRelaxedJsonEscaping);
+			values[i] = JsonSerializer.Serialize(parameter.value, JsonSchemaSerializerContext.OptionsManager.SerializerOptions);
```

Comment:
Does this mean we don't need that unsafe version of the SerializerOptions anymore?

Reply:
Or does it mean all of the usages of SerializerOptions are now unsafe? I'm not super familiar with this, but reading the documentation it seems like we probably shouldn't be using this option broadly.

Reply:
I didn't realize this was the only place this was being used.